resources/build/win/environment.inc.sh

```diff
@@ -23,8 +23,10 @@ source "$KEYMAN_ROOT/resources/build/win/environment_generated.inc.sh"
 WINDOWS_ROOT="$KEYMAN_ROOT/windows"
 WINDOWS_PROGRAM_APP="$WINDOWS_ROOT/bin/desktop"
 WINDOWS_PROGRAM_ENGINE="$WINDOWS_ROOT/bin/engine"
+WINDOWS_PROGRAM_SUPPORT="$WINDOWS_ROOT/bin/support"
 WINDOWS_DEBUGPATH_APP="$WINDOWS_ROOT/debug/desktop"
 WINDOWS_DEBUGPATH_ENGINE="$WINDOWS_ROOT/debug/engine"
+WINDOWS_DEBUGPATH_SUPPORT="$WINDOWS_ROOT/debug/suppor"
```

Comment:
I know this has merged but there is a typo missing the "t" of support
"WINDOWS_DEBUGPATH_SUPPORT="$WINDOWS_ROOT/debug/support"

Reply:
See https://github.com/keymanapp/keyman/pull/12286